src/interfaces/createTokenEvent.interface.ts

```diff
@@ -2,7 +2,7 @@ export interface CreateTokenEvent {
   _id: string
   blockNumber: number
   unixTimestamp: number
-  timestamp: Date
+  timestamp: Date | number
```

Comment:
question(blocking): Why do we need to allow a number here as well?

Reply:
@oceanByte It was initially meant to enable giving unix timestamps into the schema and mongoose would automatically convert it to Date. So Date would still have been be needed because of the DB-response object(which could also be another interface). But I removed the number here so its doesn't matter anymore.